src/main/java/com/cburch/logisim/std/ttl/AbstractTtlGate.java

```diff
@@ -221,6 +221,8 @@ protected void paintBase(InstancePainter painter, boolean drawname, boolean ghos
     var yp = y;
     var width = bds.getWidth();
     var height = bds.getHeight();
+    if (!ghost)
+      g.setColor(new Color(AppPreferences.COMPONENT_COLOR.get()));
```

Comment:
Either one-liner or multi-liner but then with mandatory brackets, please.

Reply:
And this is marked resolved yet I do not see any change?

Reply:
This got resolved a while ago, it seems that GitHub does not want to recognize this.
Take a look at the latest to commits.